src/lib/registration/widgets/remote_package_details.rb

```diff
@@ -0,0 +1,72 @@
+# Copyright (c) [2019] SUSE LLC
+#
+# All Rights Reserved.
+#
+# This program is free software; you can redistribute it and/or modify it
+# under the terms of version 2 of the GNU General Public License as published
+# by the Free Software Foundation.
+#
+# This program is distributed in the hope that it will be useful, but WITHOUT
+# ANY WARRANTY; without even the implied warranty of MERCHANTABILITY or
+# FITNESS FOR A PARTICULAR PURPOSE.  See the GNU General Public License for
+# more details.
+#
+# You should have received a copy of the GNU General Public License along
+# with this program; if not, contact SUSE LLC.
+#
+# To contact SUSE LLC about this file by physical or electronic mail, you may
+# find current contact information at www.suse.com.
+
+require "yast"
+require "cwm/common_widgets"
+
+module Registration
+  module Widgets
+    # This widgets displays the details of a remote package (name, arch, etc.)
+    class RemotePackageDetails < CWM::RichText
+      # Constructor
+      def initialize
+        textdomain("registration")
+        super
+      end
+
+      # Updates the widget's content
+      #
+      # @param package [RemotePackage] Package obtained via online search
+      def update(package)
+        lines = [
+          format(_("<b>Name:</b> %{package_name}"), package_name: package.name),
```

Comment:
All values should be HTML escaped. Although it's almost sure that e.g. `<` will not be part of the package name we should be on the safe side...

Use `h()` from `ERB::Util` like [here](https://github.com/yast/yast-registration/blob/master/src/data/registration/certificate_summary.erb#L7).

This looks quite trivial so we could built the text inline here, for more complicated cases we could use an external ERB template ([like we already do](https://github.com/yast/yast-registration/tree/master/src/data/registration)).